{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/urls.py

```diff
@@ -53,6 +53,12 @@
     from django.views import defaults as dj_default_views
     from django.urls import get_callable
 
+    # debug toolbar
+    import debug_toolbar
+    urlpatterns += [
+        url(r'^__debug__/', include(debug_toolbar.urls)),
```

Comment:
Can we use `('debug_toolbar.urls')`, just avoid one line of `import debug_toolbar` ?